src/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -80,20 +80,28 @@ public partial class HttpClientHandler : System.Net.Http.HttpMessageHandler
         public HttpClientHandler() { }
         public bool AllowAutoRedirect { get { throw null; } set { } }
         public System.Net.DecompressionMethods AutomaticDecompression { get { throw null; } set { } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
I don't understand why this attribute is getting added?  The `CheckCertificateRevocationList` property is a new property added for System.Net.Http 4.1 contract (the original 4.0 contract was original .NET Framework Desktop).

This comment applies to several other properties in the `HttpClient` class.